lib/activerecord-import/import.rb

```diff
@@ -598,6 +598,7 @@ def set_attributes_and_mark_clean(models, import_result, timestamps)
       return if models.nil?
       models -= import_result.failed_instances
       import_result.ids.each_with_index do |id, index|
+        id.tr!('()', '') if id.is_a? String
```

Comment:
[[1, 2], [3, 4]] -- that is what I was expecting to be returned. If there is a way to make that happen, that would definitely be ideal!